tests/libcxx/README.md

```diff
@@ -17,7 +17,7 @@ The tests are currently ran with the following configurations.
 
 | Compiler    | Environment          | Test set |
 |-------------|----------------------|----------|
-| clang 7.1.0 | Ubuntu 16.04 & 18.04 | Full     |
+| clang 8.0.1 | Ubuntu 16.04 & 18.04 | Full     |
```

Comment:
@BRMcLaren @rs-- Can you confirm which version of clang CI has been updated to? Is it clang 8.0.0 or 8.0.1?

Reply:
On Linux, the default clang-8 should be 8.0.0 (installed via apt). However, on Windows, only the 8.0.1 binary is available.

Reply:
Updated.